src/components/BrandConfigProvider/styles/StylesProvider.tsx

```diff
@@ -1,26 +1,25 @@
 import * as React from 'react';
-import { merge } from 'lodash';
+import merge from 'lodash/merge';
 import {
   ComponentStyles,
   RecursivePartial,
   StylesBuilder,
   NamedStyles,
   BrandConfigProviderStyles,
 } from './types';
-import { Theme } from '../theme/Theme';
-import { useTheme } from '../../../hooks/useTheme';
+import { useTheme } from '../theme/ThemeProvider';
 
 export const StylesContext = React.createContext({
-  componentStyles: {} as RecursivePartial<ComponentStyles>,
+  styles: {} as RecursivePartial<ComponentStyles>,
```

Comment:
Why this rename?

Reply:
It was part of keeping the external API of the StylesProvider clean:

```TSX
  <ThemeProvider theme={theme || {}}>
    <StylesProvider styles={styles || {}}>{children}</StylesProvider>
  </ThemeProvider>
```

From that perspective, the styles being passed in are not necessarily `componentStyles`, but a potentially other styles as well.  This leaves the door open to other style use and making the system easier to change in the future.

Getting specific is only needed on line 22 below.

The type on line 13 above might need to change if we add other types of styles, but that is easy to do without changing the external API.